source/install/upgrading-rdo-1.html.adoc

```diff
@@ -112,17 +112,66 @@ Upgrade the database schema for each service that uses the database. Run the fol
 |Image Service |glance |`# su -s /bin/sh -c "glance-manage db_sync" glance`
 |Block Storage |cinder |`# su -s /bin/sh -c "cinder-manage db sync" cinder`
 |Orchestration |heat |`# su -s /bin/sh -c "heat-manage db_sync" heat`
-|Compute |nova |
-`# su -s /bin/sh -c "nova-manage api_db sync" nova`
-
-`# su -s /bin/sh -c "nova-manage db sync" nova`
+|Compute |nova | See xref:upgrading-compute-all-at-once[Upgrading Compute (nova)].
 |Telemetry |ceilometer |`# ceilometer-dbsync`
 |Telemetry Alarming |aodh |`# aodh-dbsync`
 |Telemetry Metrics |gnocchi |`# gnocchi-upgrade`
 |Clustering |sahara | `# su -s /bin/sh -c "sahara-db-manage upgrade heads" sahara`
 |Networking |neutron |`# su -s /bin/sh -c "neutron-db-manage upgrade heads" neutron`
 |===
 
+[[upgrading-compute-all-at-once]]
+### Upgrading Compute (nova)
+
+In Ocata, the Compute service requires at least two Cells V2 cell databases, the fist database is always called `cell0` and the second database can be named arbitrarily, for example, `cell1`.
+
+Assuming that you have already created a `cell0` database with Cells V1 in your Newton environment, create and map the `cell0` database with Cells V2:
+
+```
+# su -s /bin/sh -c "nova-manage cell_v2 map_cell0" nova
+```
+
+Populate the `cell0` database by updating the `db` database schema:
+
+```
+# su -s /bin/sh -c "nova-manage db sync" nova
+```
+
+Create a `cell1` cell:
+
+```
+# su -s /bin/sh -c "nova-manage --config-file /etc/nova/nova.conf cell_v2 create_cell --name=cell1 --verbose" nova
+```
+Run this command for each cell in your environment.
```

Comment:
Run it once. Defaults to all cells in the environment unless you use the --cell_uuid param